### ISSUE TITLE
Filtrar proposições

### ISSUE THREAD

**Issue by paulot12** [interlegis/sapl#2633]
Filtrar proposições protocoladas

## Comportamento Esperado
Que seja possível filtrar as proposições apresentadas, ou seja, ao protocolar o sistema pesquisar se há alguma palavra chave nas ementas das matérias legislativas.

**Ex: Requer calçamento da Rua X, no Bairro Y.** 

**Notificar todas as matérias legislativas que indicam a Rua X, no Bairro Y.**

**Comment by edwardoliveira** [comment]
Vc quer busca textual em matéria legislativa? Isso já está disponível no SAPL desde o começo desse ano. Mas tem que instalar e habilitar.

**Comment by paulot12** [comment]
Com a pesquisa da busca textual identifica se existe alguma matéria com a ementa tratando deste assunto?

**Comment by edwardoliveira** [comment]
Sim. Funciona como as buscas no Google: vai trazer todas as matérias legislativas que tenham as palavras **Rua X, e/ou Bairro Y**
